src/BaselineOfMicrodown/BaselineOfMicrodown.class.st

```diff
@@ -44,7 +50,7 @@ BaselineOfMicrodown >> baseline: spec [
 				with: [ spec requires: #( #'Microdown-Evaluator') ];
 			
 			package: #'Microdown-HTMLExporter'
-				with: [ spec requires: #( #Microdown #'Microdown-LaTeXExporter') ];
+				with: [ spec requires: #( #Microdown #'Microdown-LaTeXExporter' 'NeoCSV') ];
```

Comment:
Why a dependency to neocsv?

Reply:
From [A tutorial on character code issues](https://web.archive.org/web/20120513165012/http://www.cs.tut.fi/~jkorpela/chars.html): In the [HTML language](https://web.archive.org/web/20120513165012/http://www.cs.tut.fi/~jkorpela/html-primer.html) one can use the notation &Auml; for [the character Ä](https://web.archive.org/web/20120513165012/http://www.cs.tut.fi/~jkorpela/latin1/3.html#C4). In the official HTML terminology, such notations are called [entity references (denoting characters)](https://web.archive.org/web/20120513165012/http://www.htmlhelp.org/reference/html40/entities/). It depends on HTML version which entities are defined, and it depends on a browser [which entities are actually supported](https://web.archive.org/web/20120513165012/http://www.htmlhelp.com/faq/html/all.html#entity-or-number).

Generated HTML contains special characters with specific format (https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references#Character_entity_references_in_HTML)[https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references#Character_entity_references_in_HTML].

The included CSV contain the mappings to convert characters to the entity references.